README.md

```diff
@@ -3,14 +3,15 @@
 
 Czech Python community homepage.
 
-|Build Status| |Test Coverage|
+[![Build Status](https://travis-ci.org/pyvec/python.cz.svg?branch=master)](https://travis-ci.org/pyvec/python.cz)
+[![Test Coverage](https://coveralls.io/repos/github/pyvec/python.cz/badge.svg?branch=master)](https://coveralls.io/github/pyvec/python.cz?branch=master)
 
 ## Installation
 
 The code is **Python 3** (the production Python version is 3.4).
 
 ```sh
-$ pip install -r requirements.txt
+$ pip install -e .
```

Comment:
`-e` is for development, while this section describes general installation, but YMMV